ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -243,7 +243,10 @@ library
                        -- Strict wrapper around SOP
                        Data.SOP.Strict
   if os(windows)
-     exposed-modules:  Ouroboros.Consensus.Storage.Seek
+     exposed-modules:
+                       -- Delete this module once all our projects have upgraded to at least
+                       -- ghc-8.10.7
+                       Ouroboros.Consensus.Storage.Seek
```

Comment:
Documenting when this can be removed so it doesn't accidentally get removed too early.

Reply:
Minor suggestion to tag this with `TODO` to make it more visible. My workflow is to periodically scan for TODO items in code.

Reply:
Done!